ChallengesWithTestsMarkVII/ChallengesSet01.cs

```diff
@@ -8,37 +8,44 @@ public class ChallengesSet01
     {
         public bool AreTwoNumbersTheSame(int num1, int num2)
         {
-            throw new NotImplementedException();
+            return (num1 == num2);          
```

Comment:
The parentheses are ok, but not required (for this one and the others).  `return num1 == num2;` is fine